### ISSUE TITLE
XML SAX parser warnings

### ISSUE THREAD

**Issue by ericvanwyk** [melt-umn/copper#1]
If there is a lexical ambiguity (and maybe in other situations as well), Copper print the following warning:

   [copper] Warning:  org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser: Property 'http://www.oracle.com/xml/jaxp/properties/entityExpansionLimit' is not recognized.
   [copper] Compiler warnings:
   [copper]   WARNING:  'org.apache.xerces.jaxp.SAXParserImpl: Property 'http://javax.xml.XMLConstants/property/accessExternalDTD' is not recognized.'

It doesn't seem to cause any problems, but we should still fix it.

**Comment by schwerdf** [comment]
I cannot reproduce this on a vanilla Copper build under OpenJDK 1.8.0_77, so I will need more information about the specific runtime environment (JRE version, classpath, *etc.*)

Specifically, it is odd that Copper should raise a warning involving the Xerces libraries, which it does not use. It seems as though Xerces is the source of the warnings -- I found several bug reports filed against the JRE and other packages, citing identical warning messages and identifying Xerces as the culprit.

**Comment by ericvanwyk** [comment]
Hi August,

On coldpress.cs I get this error when I load the 'java' module, the
'java/8' module, and the 'java/7' module.

Did you try it on logging into coldpress.cs?

Cheers,
Eric

**Comment by schwerdf** [comment]
My first test was on my own machine, but I just tried it on `coldpress.cs` as well. It worked normally right up until I loaded the `java/xerces` module, at which point it refused to run at all, throwing an `UnsupportedOperationException` from within the XML skin parser.

If you have the Xerces module loaded, you should try again with it unloaded. What is displayed on your machine when you type `echo $CLASSPATH` into the shell?

**Comment by ericvanwyk** [comment]
I've not loaded the java/xerces module. Here is what the module system
tells me is loaded:

coldpress:~ evw$ module list
Currently Loaded Modulefiles:
 1) perl/5.14.1                 9) java/8/31
 2) gnu                        10) java/maven/3.0.3
 3) local                      11) soft/gcc/4.9.2
 4) compilers/compilers        12) soft/ruby/2.2.3
 5) system                     13) soft/nodejs
 6) x11/R6.3                   14) soft/ocaml/4.01
 7) user                       15) soft/subversion/1.6-latest
 8) soft/haskell/7.10.3        16) soft/python/3.2

coldpress:~ evw$ echo $CLASSPATH
.


My classpath is just the current directory apparently.

What modules are you loading?

-Eric

**Comment by schwerdf** [comment]
I unloaded and loaded modules so the module list and classpath were exactly identical to yours and tried it again; no luck. I also tried a run using the ANT build script in the `tests` directory, having modified one of the tests to contain a lexical ambiguity; this ran without warning even with the Xerces module loaded.

This leaves, for points of variance, the specific Copper build, the specific way in which it is being invoked, and the specific input. Have you run this only through Silver, or did you try running Copper directly over the output XML?
